test/maximum-lines-in-function.js

```diff
@@ -15,7 +15,7 @@ var userConfig = {
 	}
```

Comment:
Rename this file to `test/maximum-statements-in-function.js` matching the rule name

Reply:
NVM I did it!